src/gui/TotpSetupDialog.ui

```diff
@@ -6,10 +6,16 @@
    <rect>
     <x>0</x>
     <y>0</y>
-    <width>249</width>
-    <height>278</height>
+    <width>230</width>
+    <height>240</height>
```

Comment:
Remove explicit size?

Reply:
The dialog sizes to contents on open, this is just an artifact of using Qt Designer.

Reply:
I know. I just like to keep these files as clean as possible.